src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocparagraph/Example3.java

```diff
@@ -1,53 +0,0 @@
-/*xml
-<module name="Checker">
-  <module name="TreeWalker">
-    <module name="JavadocParagraph"/>
-  </module>
-</module>
-*/
-package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocparagraph;
-
-// xdoc section -- start
-// violation 4 lines below '<p> tag should not precede HTML block-tag '<h1>''
-/**
- * No tag (ok).
- *
- * <p><h1>P tag is preceding block-tag h1, not allowed</h1>
- *
- * <p><b>P tag is preceding an inline-tag b, this is allowed</b>
- */
-public class Example3 {
-
-  // violation 4 lines below '<p> tag should not precede HTML block-tag '<ul>''
-  /**
-   * No tag (ok).
-   *
-   * <p>
-   * <ul>
-   * <li>item 1</li>
-   * <li>item 2</li>
-   * <li>item 3</li>
-   * </ul>
-   *
-   * <p>
-   * <pre>block-tag is preceded by P tag, not allowed.</pre>
-   */
-  // violation 3 lines above '<p> tag should not precede HTML block-tag '<pre>''
-  void foo1() {}
-
-  /**
-   * No tag (ok).
-   *
-   * <table>
-   * <tbody>
-   * <p>
-   * <tr>
-   * nested paragraph preceding block tag, this allowed.
-   * </tr>
-   * </tbody>
-   * </table>
-   *
-   */
-  void foo2() {}
```

Comment:
please add this example back, it is an important example for user to understand Check's working.

Reply:
I have removed the example, as the maximum file lines could be 60 only, and all the violation messages were covered in Example1 and Example2.

Reply:
@romani should we add suppression for this? or is it ok to exclude this example?

Reply:
we should have all message examples, if some are duplicated, we do not need them.
if you think that removed showing something special we can keep it. but we might need to explain user what is so special in such case.

Reply:
Okay, I will update the examples to include the Example3 case by adding the suppression.